redwood-widget-uiview-test/src/commonTest/kotlin/app/cash/redwood/widget/uiview/UIViewRedwoodViewFillWrapTest.kt

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2025 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.redwood.widget.uiview
+
+import app.cash.redwood.snapshot.testing.UIViewSnapshotCallback
+import app.cash.redwood.snapshot.testing.UIViewSnapshotter
+import app.cash.redwood.snapshot.testing.UIViewSnapshotter.Constraint.Fill
+import app.cash.redwood.snapshot.testing.UIViewSnapshotter.Constraint.Wrap
+import app.cash.redwood.snapshot.testing.UIViewTestWidgetFactory
+import app.cash.redwood.widget.AbstractRedwoodViewTest
+import app.cash.redwood.widget.RedwoodUIView
+import app.cash.redwood.widget.Widget
+import platform.UIKit.UIView
+
+/**
+ * This is like [UIViewRedwoodViewTest] but it sizes the component differently.
+ *
+ * We had a bug where `RedwoodUIView` incorrectly returned height=0 from [UIView.sizeThatFits]
+ * because its internal [platform.UIKit.UIStackView] didn't implement that operation.
+ */
+class UIViewRedwoodViewFillWrapTest(
```

Comment:
There isn’t a `WrapWrapTest` or a `WrapFillTest` yet, but that’s just cause I don’t have urgency for those.